### ISSUE TITLE
Cannot Run Two-Phase Gas-Water Problem?

### ISSUE THREAD

**Issue by OPMUSER** [OPM/opm-simulators#1649]
Declaring GAS and WATER in the RUNSPEC section and using the PVTDG and PVTW keywords in the PROPS section results in:

Program threw an exception: The specified phase configuration is not suppored.

Does this mean OPM Flow can only run in three phase mode? The commercial simulator can run this type of problem.

**Comment by totto82** [comment]
Only OIL + GAS and OIL + WATER is supported. I don't think it would be to much work to support GAS + WATER. Is this a common type of model to run?

**Comment by OPMUSER** [comment]
Yes it is for dry gas fields and we normally use the RVCONST keyword to set a constant RV, but this keyword is not implemented at the moment. The RV is just for accounting as the assumption is we are always above the dew point.
